test_runner/src/main/kotlin/ftl/args/CreateIosArgs.kt

```diff
@@ -18,8 +18,8 @@ private fun createIosArgs(
     commonArgs: CommonArgs
 ) = IosArgs(
     commonArgs = commonArgs.copy(maxTestShards = convertToShardCount(commonArgs.maxTestShards)),
-    xctestrunZip = gcloud.test!!.processFilePath("from test"),
-    xctestrunFile = gcloud.xctestrunFile!!.processFilePath("from xctestrun-file"),
+    xctestrunZip = gcloud.test?.processFilePath("from test").orEmpty(),
```

Comment:
is it team or personal preference to us isEmpty() over ?: ""

Reply:
If I right remember we don't have any team preference on it but the implementation of ```orEmpty()``` is
```

/** Returns the string if it is not `null`, or the empty string otherwise. */
@kotlin.internal.InlineOnly
public inline fun String?.orEmpty(): String = this ?: ""

``` 

in cases when we have something more after check ``` .orEmpty() ``` looks better 

```

(ArgsHelper.getDefaultProjectId() ?: "").validOrFail()

```

vs

```

ArgsHelper.getDefaultProjectId().orEmpty().validOrFail()

```

WDYT?

Reply:
I am personally more used to ?: but the VS  and use case shows a good use where it definitely looks cleaner and more reasonable to read. Thanks.